app/controllers/application_controller.rb

```diff
@@ -1,2 +1,5 @@
 class ApplicationController < ActionController::Base
+def current_user
+    @current_user = User.first
+  end
```

Comment:
- [ ] great job working on this milestone. In your PostsController, you should have new and create actions similar to the one shown below

```ruby
    def new
      @post = Post.new
    end

    def create
      @post = @current_user.posts.new(post_params)

      respond_to do |format|
        if @post.save
          format.html { redirect_to @post }
        else
          format.html { render :new }
        end
      end
    end

```ruby
    private 
    def post_params
      params.require(:post).permit(:title, :text)
    end
```
```